pyperformance/run.py

```diff
@@ -50,7 +51,28 @@ def get_run_id(python, bench=None):
     return RunID(py_id, compat_id, bench, ts)
 
 
+def get_loops_from_file(filename):
```

Comment:
I'm confused. Is this a documented pyperperformance json format? Also it would be nice if we could add this to the `.conf` parser as well.

Reply:
I don't know if it's documented, but it's the `.json` that's produced from `pyperformance run --output output.json`.

Reply:
That's a good suggestion about the `.conf` parser -- I never use it, so forgot about it.  But I'll go ahead and add support there, too.

Reply:
Oh I see. Yeah that `.json` file is definitely documented then.